npm/private/npm_import.bzl

```diff
@@ -1065,6 +1071,11 @@ def npm_import(
             This defaults to ["no-sandbox"] to limit the overhead of sandbox creation and copying the output
             TreeArtifact out of the sandbox.
 
+        lifecycle_hooks_use_default_shell_env: If True, the `use_default_shell_env` attribute of the lifecycle hooks
+            actions is set to True.
```

Comment:
This could use some kind of link to learn more about that attribute

Reply:
Yeah I'll try to find a better link there.